config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml

```diff
@@ -15,7 +16,11 @@ presubmits:
         - prow-presubmit-check
         resources:
           requests:
+            memory: "2000Mi"
             cpu: 4000m
+          limits:
+            memory: "2000Mi"
+            cpu: 4000m                    
```

Comment:
The linter is erroring on the trailing white space after this line.

`          cpu: 4000m                    ` 

```
config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml
  23:23     error    trailing spaces  (trailing-spaces) 
  ```

Reply:
Yes I have remove d the trailing space, How are you seeing that this was the error?

Reply:
It looks like there is still some trailing whitespace after the CPU. You can look at the log that the k8s-ci-robot provides or from the job itself. Here's the link to the job logs https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/29733/pull-test-infra-verify-lint/1669558900354977792. In there if you just search for "error" you'll find the offending line(s).